contexts/compound-and-earn-context.js

```diff
@@ -308,7 +308,7 @@ export function CompoundAndEarnProvider({ children }) {
           SNOBValue = SNOBHarvestable * snowballInfoQuery.data?.LastSnowballInfo?.snowballToken.pangolinPrice;
         }
 
-        if(userDepositedLP > 0 && item.name !== "xJOE"){
+        if(userDepositedLP > 0 && item.name !== "xJOE" && item.name !== "DAI.e"){
```

Comment:
shouldn't this be an or?

userDepositedLP > 0 && ( item.name !== "xJOE" || item.name !== "DAI.e" )

Reply:
this logic is right it needs to be more than 0 LP and it can't be joe nor dai